tests/redirects/test_urls.py

```diff
@@ -58,3 +58,32 @@ def test_404_url(base_url):
 def test_url(url, base_url, follow_redirects=False):
     assert_valid_url(url, base_url=base_url,
                      follow_redirects=follow_redirects)
+
+
+@pytest.mark.headless
+@pytest.mark.nondestructive
+@pytest.mark.django_db
+@pytest.mark.parametrize('url', [
+    '/en-US/',
+    '/en-US/firefox/',
+    '/en-US/firefox/all/',
+    '/en-US/firefox/new/',
+    '/en-US/firefox/mobile/',
+    '/en-US/firefox/browsers/',
+    '/en-US/firefox/products/',
+    '/en-US/firefox/accounts/',
+    '/en-US/firefox/channel/desktop/',
+    '/en-US/firefox/lockwise/',
+    '/en-US/firefox/developer/',
+    '/en-US/firefox/installer-help/',
+    '/en-US/firefox/notes/',
+    '/en-US/firefox/whatsnew/all/',
+    '/en-US/security/advisories/',
+    '/en-US/privacy/websites/',
+    '/en-US/privacy/firefox/',
+    '/credits/',
+])
+def test_real_url(url, base_url, follow_redirects=True):
+    """Ensure that major URLs work and return a 200"""
```

Comment:
Not a blocker, but this isn't really testing redirects any longer - would it make sense to move it under `/functional`?

Reply:
True. It was just similar code-wise. I can move it if you'd rather.

Reply:
Up to you as it works either way - just thought it might have caused someone confusion in the future if this ever merges :)